### ISSUE TITLE
Styling of the Admonition extension for Markdown is not correct

### ISSUE THREAD

**Issue by madrus** [squidfunk/mkdocs-material#64]
## Description

Styling of the Admonition extension for Markdown is not correct. 

### Expected behavior

These are the standard colors I see with MkDocs with the standard RTD theme:

* `note`, `seealso` - light blue
* `important`, `hint`, `tip` - green
* `warning`, `caution`, `attention` - beige-brown
* `danger`, `error` - pink

### Actual behavior

* no matter the word, everything is light blue
* only the word `warning` gives red color (instead of beige-brown)

### Steps to reproduce the bug

1. create two projects: one with MkDocs standard and one with Material.
2. `mkdocs.yml`: 
   ```yaml
   markdown_extensions: 
     - admonition
   ```
3. add this to any .md page:
   ```text
   !!! note "Explicit title within double quotes"
       Any number of other indented markdown elements.
   ```
4. play around substituting the word `note` with all of the above mentioned in both projects.

### Package versions

- Python: 2.7.12
- MkDocs: 0.16.0
- Material: 0.2.4

### System information

- OS: Windows 10
- Browser: Chrome (I have not tested with other browsers)

**Comment by squidfunk** [comment]
Already fixed in upcoming 1.0.0

**Comment by facelessuser** [comment]
Caution seems to be rendered the same color as Danger and Error in the latest pre-release.  I think it should be rendered like Warning.

**Comment by facelessuser** [comment]
Never mind.  I took a look in at the `rework` branches doc and see that you have purposely categorized the types different than the default mkdocs themes do.  I hadn't thought to look at the unpublished docs (I was still looking at the old ones) to see if the types were documented.  As they appear to be well documented and purposefully categorized as they are, I don't think there is a problem here.

**Comment by squidfunk** [comment]
It makes sense to adhere to the Readthedocs theme, as it is very widely used. So I think it's a good point you made, because it minimizes switching cost.

**Comment by squidfunk** [comment]
The RTD admonition functionality is now completely mirrored + the ones Material adds (e.g. `summary`, `bug` etc). Fixed in 45208dea733959d71b46d5fb7f7e9d045a414b8d
